### PR TITLE
Grid text list

### PR DESCRIPTION
**Pull request by Sahil25061999**
added:
-grid
-typography
-list styles

### CODE REVIEW

**Comment by swstica** [comment]
1. I think the whole website needs styling.
2. Colours used in the grid are a little bright, subtle colours would be better, ig.

File: component/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div class="grid-item"></div>
+      <div class="grid-item"></div>
+      <div class="grid-item"></div>
+    </div>
+    <br />
+
+    <h2 class="text-center">Grid-sidebar</h2>
+    <div
+      class="grid-temp-container grid-container grid-2-sidebar primary-color"
+    >
```
Comment (pradipgharge) [inline component/grid/grid.html:30]: Check Indentation here

File: component/text-utility/text.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="xlarge-text strike-text">Strike through text</div>
+
+    <br />
+    <div class="medium-text underline-text">This text has undeline</div>
+    <br />
+    <div class="medium-text primary-text-color">
+      This text has primary text color.
+    </div>
+    <div class="medium-text secondary-text-color">
+      This text has secondary text co lor.
```
Comment (pradipgharge) [inline component/text-utility/text.html]: Check spacing in color

**Review by pradipgharge — COMMENTED** [review]
As swastika said, styling needs to be improved. Rest code looks good to me.

**Review by manish-GH — COMMENTED** [review]
Your individual components look great, but the page as a whole looks clumped. You could consider centering everything instead of having it at the left corner and adding a position sticky to the navbar which has the component links

**Pull request merged**